pylint/lint.py

```diff
@@ -1262,6 +1262,14 @@ def __init__(self, args, reporter=None, exit=True):
                        'checker will be displayed'}),
 
             ), option_groups=self.option_groups, pylintrc=self._rcfile)
+
```

Comment:
I think that the order is relevant here, as the linter has not been fully configured when the configuration is loaded, although out of the top of my head, I can't pinpoint the exact reasons. I *think* the CI will pass once this is reverted. If you have some time, can you check why this is happening?